docs/releasenotes/8.2.0.rst

```diff
@@ -21,10 +21,17 @@ TODO
 API Additions
 =============
 
-TODO
-^^^^
+ImageShow.IPythonViewer
+^^^^^^^^^^^^^^^^^^^^^^^
 
-TODO
+If IPython is present, this new ``ImageShow.Viewer`` subclass will be
+registered. It displays images on all IPython frontends. This will be helpful
+to users of Google Colab, allowing ``im.show()`` to display images.
+
+It is lower in priority than the other default Viewer instances, so it will
```

Comment:
> It is lower in priority than the other default Viewer instances

Please correct me if I'm wrong, but this is false. The new viewer has the highest priority AFAICT

Reply:
You're right! Thanks for catching that. However, the documentation correctly states my intention, so I've updated the code to match it.